src/common/eeprom.cpp

```diff
@@ -195,7 +197,8 @@ static const eeprom_vars_t eeprom_var_defaults = {
     {"Custom2", FLT_MAX },
     {"Custom3", FLT_MAX },
     {"Custom4", FLT_MAX },
-	0,               // EEVAR_SELFTEST_RESULT
+	  0,               // EEVAR_SELFTEST_RESULT
```

Comment:
Extra tab here 🧐

Reply:
clang format is off at this point :(

Reply:
there are some edge cases of clang-format, I wouldn't block the PR because of this